src/main/java/org/opentripplanner/graph_builder/linking/SimpleStreetSplitter.java

```diff
@@ -69,6 +78,10 @@
 
     public static final int MAX_SEARCH_RADIUS_METERS = 1000;
 
+    public Boolean skipVisibilty = true;
```

Comment:
Do not have public mutable fields.  Use getters/setters.

Reply:
Also, this field is only used in a negated way, so why not rename it and switch its polarity?